docs/sources/v0.50.x/set-up/set-up-distributed-k6/install-k6-operator.md

```diff
@@ -0,0 +1,100 @@
+---
+weight: 100
+title: Install k6-operator
+---
+
+# Install k6-operator
+
+This guide provides step-by-step instructions on how to install k6 operator.
+
+## Before you begin
+
+To install k6 operator, you'll need:
+
+- A Kubernetes cluster, along with access to it.
+- [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl).
+
+## Deploy the operator
+
+There are three different options that you can use to deploy the k6-operator.
+
+### Deploy with bundle
+
+The easiest way to install the operator is with bundle:
+
+```bash
+curl https://raw.githubusercontent.com/grafana/k6-operator/main/bundle.yaml | kubectl apply -f -
+```
+
+Bundle includes default manifests for k6-operator, including a `k6-operator-system` namespace and k6-operator deployment with the latest tagged Docker image. Customizations can be made on top of this manifest as needed, for example, with `kustomize`.
+
+### Deploy with Helm
+
+Helm releases of k6-operator are published together with other Grafana Helm charts. You can install it with the following commands:
+
+```bash
+helm repo add grafana https://grafana.github.io/helm-charts
+helm repo update
+helm install k6-operator grafana/k6-operator
+```
+
+You can also pass additional configuration options with a `values.yaml` file:
+
+```bash
+helm install k6-operator grafana/k6-operator -f values.yaml
+```
+
+Refer to the [k6-operator samples folder](https://github.com/grafana/k6-operator/blob/main/charts/k6-operator/samples/customAnnotationsAndLabels.yaml) for an example file.
+
+You can find a complete list of Helm options in the [k6 operator charts folder](https://github.com/grafana/k6-operator/blob/main/charts/k6-operator/README.md).
+
+### Deploy with Makefile
+
+In order to install the operator with a Makefile, you'll need:
+
+- [go](https://go.dev/doc/install)
+- [kustomize](https://kubectl.docs.kubernetes.io/installation/kustomize/)
+
+A more manual, low-level way to install the k6 operator is by running the command below:
+
+```bash
+make deploy
+```
+
+This method may be more useful for development of the k6-operator, depending on specifics of the setup.
+
+## Install the CRD
+
+The k6-operator includes custom resources called `TestRun`, `PrivateLoadZone`, and `K6`. They're automatically installed when you do a deployment or install a bundle, but you can also manually install them by running:
+
+```bash
+make install
+```
+
+{{< admonition type="warning" >}}
+
+The `K6` CRD has been replaced by the `TestRun` CRD and will be deprecated in the future. We recommend using the `TestRun` CRD.
+
+{{< /admonition >}}
+
+## Deploy with custom namespace
```

Comment:
@heitortsergent, regarding this heading: I now think it could also be a "Watch namespace(s)" or "Watching namespace(s)". So like env var itself :smile: 
Rationale: `WATCH_NAMESPACE` is a "known" concept among at least some users of k8s operators (e.g. [here](https://kubernetes.github.io/ingress-nginx/user-guide/cli-arguments/)), i.e. it is not unique to k6-operator, so it should be both valid and understandable to use in heading.

Then, "namespaced deployment" would mean deploying k6-operator confined to a namespace - that's what I described during our chat as a "good to have" guide in the future. (Btw, the issue for this guide is basically [here](https://github.com/grafana/k6-operator/issues/150))

WDYT?

Reply:
That makes sense to me @yorugac! I updated the heading to "Watch namespace". 🙇